### PR TITLE
Feat/decision card story

### PR DESCRIPTION
**Pull request by dannyk3941**
#### What's this PR do?
- Create Decision Cards patterns and stories
- Create margin, padding, font-weight utility classes from css-variables
- Unminify build files (css and js) for viewing and development
#### Where should the reviewer start?
- `npm install`
- `npm run styles-build`
#### How should this be manually tested?
- Check "Card" stories and test controls.

### CODE REVIEW

File: packages/styles/vite.config.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default defineConfig({
   },
   plugins: [],
   build: {
+    minify: false,
```
Comment (dgibson666) [inline packages/styles/vite.config.js:14]: Why not minify? The browser and VSCode can both reformat it as readable. Does this affect something else?

**Review by dgibson666 — COMMENTED** [review]
A few notes in addition to the code-level ones:

* Without a generic card, it's hard to see how this "decision card" differs or can leverage generic card styles, which I think it should. It seems weird to put a `cbp-card-decision` class on the main parent and use that naming all the way down if it may only impact part of the design. Can we make this a modifier class in addition to `cbp-card`?
* Remember that the browser-default for `button` tags is `type="submit"`, so we should be explicitly specifying `type="button"` on all button controls. They may work fine in Storybook demos, but once an app uses our components inside of a `form` tag, they will wreak havoc.

File: packages/styles/src/sass/components/card/_decision.scss [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+.cbp-card-decision__actions {
+  display: flex;
+
+  & > button,
+  & > a[href] {
+    @extend .cbp-btn;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    height: 44px;
```
Comment (dgibson666) [inline packages/styles/src/sass/components/card/_decision.scss:24]: I missed this the first time through. Let's not add a fixed height; add padding to get the desired height instead.

Comment (dannyk3941) [reply packages/styles/src/sass/components/card/_decision.scss]: Removed the fixed height from `_button.scss` but that whole thing needs a refactor and added the correct padding under `.cbp-card__actions`

File: packages/styles/src/sass/components/card/_base.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  display: flex;
+  justify-content: space-between;
+}
+
+.cbp-card__title {
+  color: var(--cbp-color-text-darker);
+  font-weight: var(--cbp-font-weight-medium);
+  font-size: var(--cbp-font-size-heading-lg);
+  line-height: var(--cbp-line-height-sm);
+  letter-spacing: 0.12px;
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/sass/components/card/_base.scss]: Removed

File: packages/styles/vite.config.js (hunk 1 above)
Comment (dannyk3941) [reply packages/styles/vite.config.js]: Didn't even think about that haha

File: packages/styles/src/sass/components/card/_decision.scss [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+/* DANGER */
+.cbp-card-decision--danger {
+  & .cbp-card__title {
+    color: var(--cbp-color-danger-base);
+  }
+
+  & .cbp-card-decision__content {
+    background-color: var(--cbp-color-danger-lighter);
+    border: 2px solid var(--cbp-color-danger-base);
+    border-bottom: 0;
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/sass/components/card/_decision.scss:43]: Replaced

File: packages/styles/src/sass/components/card/_decision.scss [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+      border-top-right-radius: 0px;
+      border-bottom-left-radius: 0px;
+      border-bottom-right-radius: 0px;
+    }
+
+    & > button:last-child,
+    & > a[href]:last-child {
+      border-bottom-right-radius: 5px;
+      border-bottom-left-radius: 0px;
+    }
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/sass/components/card/_decision.scss]: Thank you for the suggestion!

File: packages/styles/src/sass/components/card/_decision.scss [hunk 6]
```diff
@@ -0,0 +1,127 @@
+@use '../button/index' as *;
+
+.cbp-card-decision {
+  min-width: 312px;
```
[the remark this answers is not in the document]
Comment (dannyk3941) [reply packages/styles/src/sass/components/card/_decision.scss]: Removed and added.

**Review by dgibson666 — APPROVED** [review]
looks great!

**Pull request merged**